tests/test_fix_seed/test.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#set -e
+
+Check_difference() {
+    diff -q $1 $2
+    if [[ $? == 0 ]]; then
+	echo "error: files must be different"
+	exit 1
+    fi
+}
+
+Check_similarity() {
+    diff -s $1 $2
+    if [[ $? == 1 ]]; then
+	echo "error: files must be the same"
+	exit 1
+    fi
+}
+
+Check() {
+    PREC=$1
+    MODE="MCA"
+
+    for BACKEND in "mca" "mca_mpfr"; do
+	echo -e "\nChecking backend ${BACKEND} at PRECISION $PREC MODE $MODE"
+
+	BACKENDSO="libinterflop_${BACKEND}.so"
+
+	out_seed_1="out_seed_${BACKEND}.1"
+	out_seed_2="out_seed_${BACKEND}.2"
+
+	out_seed_1="out_seed_${BACKEND}.1"
+	out_seed_2="out_seed_${BACKEND}.2"
+
+	rm -f out_seed_{1,2} out_no_seed_{1,2}
+	
+	export VFC_BACKENDS="${BACKENDSO} --precision $PREC --mode $MODE"
+	./test > out_no_seed_1
+	./test > out_no_seed_2
+	Check_difference out_no_seed_1 out_no_seed_2
+	
+
+	export VFC_BACKENDS="${BACKENDSO} --precision $PREC --mode $MODE --seed=0"
+	./test > out_seed_1
+	./test > out_seed_2
+	Check_similarity out_seed_1 out_seed_2
+	    
+    done
+}
+
+for op in "+" ; do
```

Comment:
Since we are only checking for op +, could we simplify the test ? test.c seems overly complicated for what we are testing.
Just a simple iteration that accumulates 100 times the value .01 should be enough.